xbmc/storage/MediaManager.cpp

```diff
@@ -626,20 +622,23 @@ void CMediaManager::EjectTray( const bool bEject, const char cDriveLetter )
 void CMediaManager::CloseTray(const char cDriveLetter)
 {
 #ifdef HAS_DVD_DRIVE
-#if defined(TARGET_DARWIN)
-  // FIXME...
-#elif defined(TARGET_FREEBSD)
-  // NYI
-#elif defined(TARGET_POSIX)
-  char* dvdDevice = CLibcdio::GetInstance()->GetDeviceFileName();
-  if (strlen(dvdDevice) != 0)
+#if defined(TARGET_POSIX)
+  std::shared_ptr<CLibcdio> c_cdio = CLibcdio::GetInstance();
```

Comment:
`c_cdio` ??? What's the reason for the prefix?

Reply:
all the code in the same file uses this naming, this is temporary and I intend to remove it from this file and split per platform. But adjusted anyway